### PR TITLE
Remove deprecate Storage.create()

### PR DESCRIPTION
**Pull request by frankyn**
Remove deprecated method with an undeprecated method.

### CODE REVIEW

File: bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java [hunk 1]
```diff
@@ -54,22 +56,33 @@
...
     DateTimeFormatter dtf = DateTimeFormat.forPattern("-YYYY-MM-dd-HHmmssSSS");
     DateTime dt = DateTime.now(DateTimeZone.UTC);
     String dtString = dt.toString(dtf);
     final String fileName = filePart.getSubmittedFileName() + dtString;
-
-    // the inputstream is closed by default, so we don't need to close it here
+    
+    // The InputStream is closed by default, so we don't need to close it here
+    // Read InputStream into a ByteArrayOutputStream.
+    InputStream is = filePart.getInputStream();
```
Comment (kurtisvg) [inline bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java]: I think this all can be done more simply: 
```java
FileInputStream file = new FileInputStream(fileName);
byte[] fileContent = Files.readAllBytes(file.toPath());
```

Then pass fileContent below.

File: bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java [hunk 2]
```diff
@@ -54,22 +56,33 @@
    * Uploads a file to Google Cloud Storage to the bucket specified in the BUCKET_NAME
    * environment variable, appending a timestamp to end of the uploaded filename.
    */
-  @SuppressWarnings("deprecation")
+  // Note:This sample assumes small files are uploaded. For large files or streams use:
```
Comment (kurtisvg) [inline bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java]:
```suggestion
  // Note: this sample assumes small files are being uploaded. For large files or streams use 
```

File: bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java (hunk 1 above)
Comment (frankyn) [reply bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java]: thanks! much simpler.

**Comment by kurtisvg** [comment]
```
------------------------------------------------------------
- testing bookshelf/3-binary-data
------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[ERROR] /tmpfs/src/github/getting-started-java/bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java:[28,26] error: ';' expected
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project bookshelf-3: Compilation failure
[ERROR] /tmpfs/src/github/getting-started-java/bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java:[28,26] error: ';' expected
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

 Tests failed. 
```

File: bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java (hunk 1 above)
Comment (frankyn) [reply bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java]: Hey @kurtisvg, 

I jumped to quickly to add this code in.

The servlet handler doesn't provide the filePath and instead provides `filePart.getInputStream()` to access the uploaded data. I don't think this simplification will work in this case. 

Could you provide more context on this? I may have missed something but it not be usable in this case.

Comment (kurtisvg) [reply bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java]: I was looking at L65, which looked like it was getting the filename. I missed it was pulling from a Servlet stream. 

It looks like in Java 9, there is `InputStream.readAllBytes` but I'm guessing you still want to support Java 8. In that case, you'll probably have to do something like this: https://stackoverflow.com/a/1264737/9414803 (which you may already have been doing)

Comment (frankyn) [reply bookshelf/3-binary-data/src/main/java/com/example/getstarted/util/CloudStorageHelper.java]: Thanks @kurtisvg! +1 I saw Java 9 had a special helper, maybe in the future we can use it. For now, I'll revert and try to get Kokoro to pass.

**Pull request merged**